packages/terra-date-time-picker/src/terra-dev-site/doc/example/DateTimePickerDefaultDateTime.jsx

```diff
@@ -18,7 +19,14 @@ const defaultProps = {
 class DateTimePickerExample extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { dateTime: this.props.value };
+    let dateTimeDisplay = props.value;
+    const dateTime = DateTimeUtils.createSafeDate(dateTimeDisplay);
+
+    if (dateTime && dateTime.isValid()) {
+      dateTimeDisplay = dateTime.format();
+    }
+
+    this.state = { dateTime: dateTimeDisplay };
```

Comment:
I don't think you need these changes in this specific example. The value prop set below using `moment().format()` would never be the missing hour.

Reply:
yes, that's right. reverted the code changes : 4aaa072145241a931da717a591d0a5f9e644356f